.travis.yml

```diff
@@ -5,18 +5,22 @@ matrix:
   include:
   - python: 3.7
     dist: xenial
+    env: TF=2
   - python: 3.6
     dist: xenial
+    env: TF=2
   - python: 3.5
     dist: xenial
+    env: TF=1
 env:
   global:
   - COVERALLS_PARALLEL=true
   - LOGGING=info
 install:
 - pip install -r requirements.txt
-- pip install wheel pytest pytest-cov --upgrade
-- pip install tensorflow==1.13.1
+- pip install wheel pytest pytest-cov absl-py --upgrade
```

Comment:
The test suite seems to run successfully without installing "absl-py". Maybe it would be nice to have a test case where in the example an error would arise without having absl-py installed.

Reply:
If you don't have TensorFlow installed, the test suite runs fine (without `absl-py`). However, installing TensorFlow drags in `absl-py` as a dependency, which then breaks pytest.

The latest version of `absl-py` (v0.8) breaks pytest slightly less than v0.7.1, though, which is nice!

The only reason it is added here is that TravisCI has a habit of caching python wheels, and not upgrading packages unless you request it.